HW_1/Task4.py

```diff
@@ -0,0 +1,15 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+# *Пример:*
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n = int(input("Введите n: "))
+m = int(input("Введите m: "))
+k = int(input("Введите k: "))
+
+if(k%n == 0 and k%n < m):
+    print('yes')
+elif(k%m == 0 and k%m < n):
```

Comment:
Во-первых, Михаил, скобки тут не нужны. Во-вторых, как вы можете заметить, правое условия в каждом условном операторе одинаковое. Значит, по этому условию можно объединить 2 оставшихся и поместить под один `if`. Я этого делать не буду, исправлю только пробелы, но вы сделайте это самостоятельно.
```suggestion
if k % n == 0 and k % n < m:
    print('yes')
elif k % m == 0 and k % m < n:
```